containers/Makefile

```diff
@@ -79,7 +89,22 @@ verifier: build/ $(VERIFIER_SOURCE) build/repositories/golem/ build/verifier/gol
 	docker build --tag $(IMAGE_PREFIX)verifier:$(IMAGE_VERSION) build/verifier/
 	docker tag $(IMAGE_PREFIX)verifier:$(IMAGE_VERSION) $(IMAGE_PREFIX)verifier:latest
 
-docker-clean: nginx-proxy-clean concent-api-clean postgresql-clean verifier-clean
+signing-service: build/ $(CONCENT_API_SOURCE) $(SIGNING_SERVICE_SOURCE) build/signing-service/signing-service/
+	docker build --tag $(IMAGE_PREFIX)signing-service:$(IMAGE_VERSION) build/signing-service/
+	docker tag $(IMAGE_PREFIX)signing-service:$(IMAGE_VERSION) $(IMAGE_PREFIX)signing-service:latest
+
+concent-signing-service-package: build/ $(CONCENT_API_SOURCE) $(SIGNING_SERVICE_SOURCE) build/concent-signing-service-%.tar.xz
+
+build/concent-signing-service-%.tar.xz:  build/signing-service/signing-service/
```

Comment:
This should be somewhere where it will be apparent that it's a final product, not an intermediate file. I'd create a `dist/` directory next to `build/`.

Reply:
Done.